1-exercises/A-array-find/exercise.js

```diff
@@ -15,9 +15,10 @@ let names = [
   "Anastasia",
   "Karim",
   "Ahmed",
-];
+]; 
 
 let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+  return findLongNameThatStartsWithA = names.find( names.length > 7 && startsWith("A"));
 
```

Comment:
good one. Just mind that startsWith() is not supported in IE 11 (and earlier versions)